wp-config-pantheon.php

```diff
@@ -96,6 +96,13 @@
  * To override, define your constant in your wp-config.php before wp-config-pantheon.php is required.
  */
 
+if ( ! defined('PANTHEON_HOSTNAME' ) ) {
+    $site_name = $_ENV['PANTHEON_SITE_NAME'];
+    $hostname = ! isset( $_SERVER['HTTP_HOST'] ) ? $_ENV['PANTHEON_ENVIRONMENT'] . "-$site_name.pantheonsite.io" : $_SERVER['HTTP_HOST'];
+    $hostname = $_ENV['PANTHEON_ENVIRONMENT'] === 'lando' ? "$site_name.lndo.site" : $hostname;
```

Comment:
Somewhere else we recently opted to use a different patten to identify when we are on Lando (since `lando` could be a multidiev name). Don't recall what you did, but probably good to do that again.

Reply:
I couldn't remember what I did, either, but `lando` is for sure the `$_ENV['PANTHEON_ENVIRONMENT']` value when using it. I'll see if I can track it down, though.